File: src/HostelSystem.Domain/Rezerwacja.cs
using System;
using System.Collections.Generic;

namespace HostelSystem.Domain
{
    public class Rezerwacja : CommonEntity
    {
        public string KodRezerwacji { get; set; }
        public DateTime DataUtworzenia { get; set; }
        public decimal Cena { get; set; }
        public DateTime DataZameldowania { get; set; }
        public DateTime DataWymeldowania { get; set; }
        public decimal Prowizja { get; set; }

        public ICollection<Gosc> Goscie { get; set; }
    }
}

File: src/HostelSystem.Web/Sencha/app/model/Rezerwacja.js
Ext.define('HostelSystem.Web.model.Rezerwacja', {
    extend: 'Ext.data.Model',

    fields: ['kodRezerwacji', 'dataUtworzenia', 'cena', 'dataZameldowania', 'dataWymeldowania', 'prowizja', 'id'],

    hasMany: { model: 'HostelSystem.Web.model.Gosc', name: 'goscie' }
});